pkg/client/create_builder_test.go

```diff
@@ -1079,11 +1081,11 @@ func testCreateBuilder(t *testing.T, when spec.G, it spec.S) {
 					h.AssertEq(t, len(layers), 1)
 				})
 
-				when("with exclude", func() {
+				when("only some modules are flattened", func() {
 					it("creates 1 layer for buildpacks and 1 layer for buildpack excluded", func() {
 						prepareFetcherWithRunImages()
-						opts.Flatten = true
-						opts.FlattenExclude = []string{"flatten/bp-7@7"}
+						opts.Flatten, err = buildpack.ParseFlattenBuildModules([]string{"flatten/bp-1@1,flatten/bp-2@2,flatten/bp-4@4,flatten/bp-6@6,flatten/bp-3@3,flatten/bp-5@5"})
```

Comment:
I would love to also see a test where the array has two elements. In this case we want to assert that there are 3 layers, no?

Reply:
Yeah, I will add more test coverage

Reply:
each array element represents a layer, let's say ["1,2,3", "4,5,6"] in this case, yes we should have 3 layers. 
- layer1 = [1,2,3]
- layer2 = [4,5,6]
- layer3 = [7]